lib/active_record_florder/configurable.rb

```diff
@@ -48,5 +52,9 @@ def min_position_delta
     def next_position_step
       self.class.next_position_step
     end
+
+    def move_populate_all
```

Comment:
Well I'm not 100% sure that this naming is clean. If you have any suggestion please send it.

I use `populate` in public api because it keep it simple to use this. On another hand internally `move_populate_all` is used in instances to prevent conflicts with other possible values (remember that this is all "meta coding" which extends Models instances itself and we have not clue which attributes and methods they can have).